docs/user-manual/en/resource-adapter.md

```diff
@@ -0,0 +1,130 @@
+# Resource Adapter
+
+For using Apache ActiveMQ Artemis in a JavaEE/JakartaEE environment, you can use the ResourceAdapter. Before you 
+start, read carefully the [JMS chapter](using-jms.md) 
+
+##In which case do you have to use a RA and in which not?
+The main reason is the requirement of 
+[XA](https://jakarta.ee/specifications/transactions/2.0/jakarta-transactions-spec-2.0.html), distributed 
+transactions over multiple resources. If it is important for you no message get lost and multiple resources (e.g. 
+database and messaging) must be in sync, there is no way around it.
+
+##Versions
+let's give you a short overview of the versions, to be sure you pick the correct *.rar.
+
+| | artemis-ra-rar   |     |     |
+| --------- | ------ | --- | --- |
+| Artemis   | JavaEE | JCA | JMS |
+| \>= 2.18.0 | \>=8      | 1.7 | 2.0 |
+| <= 2.17.0 | 7      | 1.5 | 2.0 |
+
+| | artemis-jakarta-ra-rar | | |
+| --- | ------- | --- | --- |
+| Artemis | JavaEE | JCA | JMS |
+| \>= 2.18.0 | \>=9 | 2.0 | 3.0 |
```

Comment:
The doc is version specific, I dont think it should concern itself with specifying stuff for what are even now already old Artemis versions. I would remove it or change it so it just describes the difference between the current rars, e.g

artemis-ra-rar A B C
artemis-jakarta-ra-rar X Y Z